cli/command/container/stats_helpers.go

```diff
@@ -164,6 +169,65 @@ func collect(ctx context.Context, s *Stats, cli client.APIClient, streamStats bo
 	}
 }
 
+func formatCPUAutoRange(ar map[string]string) string {
+	numCPU, exist := ar["numCPU"]
+	if !exist {
+		numCPU = "--"
+	}
+
+	percentOpti, exist := ar["percentOpti"]
+	if !exist {
+		percentOpti = "--"
+	}
+
+	usageOpti, exist := ar["usageOpti"]
+	if !exist {
+		usageOpti = "--"
+	}
+
+	return fmt.Sprintf("%s / %s%% / %s", numCPU, percentOpti, usageOpti)
+}
+
+func formatMemoryAutoRange(ar map[string]string) string {
+	newMinRange, exist := ar["nmin"]
+	if !exist {
+		newMinRange = "--"
+	}
+
+	newMaxRange, exist := ar["nmax"]
+	if !exist {
+		newMaxRange = "--"
+	}
+
+	sugMin, exist := ar["sugmin"]
+	if !exist {
+		sugMin = "--"
+	}
+
+	sugMax, exist := ar["sugmax"]
+	if !exist {
+		sugMax = "--"
+	}
+
+	return fmt.Sprintf("%s/%s  %s/%s | ", addUnit(newMinRange), addUnit(newMaxRange), addUnit(sugMin), addUnit(sugMax))
+}
+
+func addUnit(value string) string {
+	lval := len(value)
+
+	if lval > 9 {
```

Comment:
I think there's already a vendored library for that. Can you check `github.com/go-units/size.go` ? 🤔